frontend/src/pages/Form.js

```diff
@@ -1,19 +1,16 @@
 import React, { useEffect, useState, useRef } from 'react';
-import jwt_decode from 'jwt-decode';
```

Comment:
I think we can get rid of this from package.json now

Reply:
Sure!